assets/css/quick-filter.css

```diff
@@ -24,18 +24,8 @@
   height: 1.25rem;
 }
 
-.quick-filter.selected {
-  border-radius: 0.5rem;
-  background: var(--primary-900, #0f1815) !important;
-  border-color: var(--primary-900, #0f1815) !important;
-}
-
-.quick-filter.selected .quick-filter__name {
-  color: var(--white) !important;
-}
-
 .quick-filter.selected .quick-filter__icon {
-  background-color: var(--white) !important;
```

Comment:
!important property was not required. Moved these styles after `:hover` selector